src/move/move.ts

```diff
@@ -70,6 +70,40 @@ const checkMoveAfterFirstTurn = (
   };
 };
 
+export const getPlayableTilesForNextMove = (
+  board: Board,
+  lastPlayedTile: Tile
+): Tile[] => {
+  let tiles: Tile[] = [];
+
+  board.forEach((line) => {
+    line.forEach((tile) => {
+      if (
+        tile == null ||
+        tile.symbol === NEUTRALE_TILE.symbol ||
+        tile.playedBy != null
+      ) {
+        return;
+      }
+
+      const isSymbolConstraintUnrespected =
+        lastPlayedTile.symbol != tile.symbol;
+      const isColorConstraintUnrespected = lastPlayedTile.color != tile.color;
+
+      if (
+        tile.lastPlayed ||
+        (isSymbolConstraintUnrespected && isColorConstraintUnrespected)
+      ) {
+        return;
+      }
```

Comment:
Why not reuse the `checkMoveAfterFirstTurn` fn?

Reply:
Done !

Reply:
Yeah, François even has a saying: "never repeat yourself" (in the code).
This is exactly that! :slightly_smiling_face: